CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+contributions via GitHub pull requests. This document outlines some of the
+conventions on development workflow, commit message formatting, contact points,
+and other resources to make it easier to get your contribution accepted.
+
+
+## How to Contribute
+
+Pull Requests (PRs) are the main and exclusive way to contribute to the project.
+In order for a PR to be accepted it needs to pass the personal evaluation of at least one of the [maintainers](MAINTAINERS).
+
+
+### Format of the commit message
+
+Every commit message should describe what was changed and, if applicable, the GitHub issue it relates to:
+
+```
+Skip argument validations for unknown capabilities. Fixes #623
+```
+
+The format can be described more formally as follows:
+
+```
+<what changed>. [Fixes #<issue-number>]
+```
+
+
+# Architecture
+
+The landing contains 3 parts:
+
+- The [static site](#static-site) itself: a webpack app,
+- a [Go API](#api) to serve the latest blog posts, job offers and proxy to slackin,
+- [Slackin](#slackin) to send invitations to our visitors to join our slack community.
+
+In production, each part lives in a separated container.
+
+
+## Static site
+
+The static site is built with:
+- **webpack**: `javascript` (with some `REACT` components) and `css` (from `SASS` stylesheets)
+- **hugo**: for the `HTML` from `Go` templates and `YAML` data
+
+Its source code is under:
+- [`hugo` directory](hugo): for `hugo` sources:
+  - `content` directory: contains one `.md` per landing page that defines each page metadata,
+  - `data` directory: contains the landing variable content like titles, captions, descriptions, projects...
+  - `layout` directory: contains the `HTML` templates.
+- [`src` directory](src): for `js` and `sass` code,
+- [`static/img` directory](static/img): for site images,
+
+
+## API
+
+The API serves (a cached version of):
+- the 3 latest blog posts tagged as `technical`, and the 3 latest blog posts tagged as `culture` for the home page,
+- all the opened positions at Lever,
+
+Handles the visitor requests to join our slack community, sending an slack invitation through [Slackin](#slackin)
+
+Its source code is under [`api` directory](api).
+
+
+## Slackin
+
+The slackin container listen for invitation requests made from the landing [api](#api).
+
+For [technical reasons](https://github.com/src-d/landing/issues/62#issuecomment-327194704), slackin container is built from the release [v0.13.1](https://github.com/rauchg/slackin/tree/0.13.1)
+
+
+# Development
+
+
+## Requirements
+
+You should already have [Go installed](https://golang.org/doc/install#install), and properly [configured the $GOPATH](https://github.com/golang/go/wiki/SettingGOPATH)
+```
+go version; # prints your go version
+echo $GOPATH; # prints your $GOPATH path
+```
+
+The project must be under the `$GOPATH`, following the Go import conventions, what means you can go to its directory running:
+```
+cd $GOPATH/src/github.com/src-d/landing
```

Comment:
should it be installed by `go get` or we need to `git clone` repository in that directory?

Reply:
since the landing api has some dependencies, it will be better if it is downloaded/installed using
```shell
go get github.com/src-d/landing/...
```
I'll add it